### ISSUE TITLE
Disable single FAN sensor??

### ISSUE THREAD

**Issue by jsergio123** [arnuschky/dell-bmc-firmware#4]
Do you think it would be possible to disable the monitoring of a single fan completely? It seems the FAN 1 RPM sensor has failed and is reporting 0 RPM. I have tried swapping the fans around and setting the value to 0 but it had no effect. I would really hate to have to swap the entire board for just a fan sensor. The thing sounds like a jet right now due to losing fan redundancy.

**Comment by arnuschky** [comment]
I don't think that this is possible, but to be entirely honest I don't remember enough about this project to be certain. It's been a few years - sorry!

**Comment by jsergio123** [comment]
Thank you for the reply.
